### DISCUSSION TITLE
add a tangential force BC

### DISCUSSION THREAD

**Discussion by lyyc199586** [idaholab/moose#24726] [Q&A Modules: Solid mechanics]
so I have a inclined boundary and I want to enforce both normal and tangential force on that boundary, there is `Pressure` to enforece normal one, but the tangential need some extra work.

![null](https://github.com/idaholab/moose/assets/20653957/053b3306-ccc3-42ed-b297-66134f52e6c3)

I would expect I just need to modifiy some code in `Pressure` BC

**Comment by lyyc199586** [comment]
@GiudGiud

**Reply by GiudGiud** [reply]
@hugary1995  do you want to add to what you explained here to another OP on a related topic
https://github.com/idaholab/moose/discussions/23377#discussioncomment-4905519
I dont know if there's been additional developments on this over the last 4 months

**Reply by hugary1995** [reply]
Can you be more specific about what issue you are running into while implementing this BC? If you are just requesting a new feature, please open an issue.

**Comment by lyyc199586** [comment]
I opend an issue about it #24743
